cmd/chunks-inspect/loki.go

```diff
@@ -1,97 +1,18 @@
 package main
 
 import (
-	"bytes"
-	"compress/gzip"
 	"encoding/binary"
 	"fmt"
-	"hash/crc32"
+	"github.com/grafana/loki/pkg/chunkenc"
```

Comment:
I think we also have a lint rule that will say that this needs to be at the bottom of the import list with a newline in between it and the stdlib imports.